orc8r/tools/fab/vagrant.py

```diff
@@ -21,8 +22,14 @@ def __ensure_in_vagrant_dir():
     """
     pwd = local('pwd', capture=True)
     if not os.path.isfile(pwd + '/Vagrantfile'):
-        print("Error: Vagrantfile not found. Try executing from fbcode/magma")
-        exit(1)
+        # check if we are on a vagrant subdirectory
+        with cd(pwd):
+            if not local('Vagrant validate', capture=True):
+                print(
+                    "Error: Vagrantfile not found. "
+                    "Try executing from fbcode/magma",
```

Comment:
The dir path `fbcode/magma` doesn't exist any more. Is this referring to the MAGMA_ROOT?

Reply:
We can remove that message. I guess it was there from before and was never modified